onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1148,11 +1130,20 @@ def test_output_match(self, device: str, dtype: torch.dtype, op):
                     input_onnx, kwargs_onnx = input_wrangler(input_onnx, kwargs_onnx)
                 torch_output = op(*inputs, **cpu_sample.kwargs)
 
-                # TODO: add pytree structure comparison.
                 flattened_torch_outputs, _ = pytree.tree_flatten(torch_output)
+                if op.name.startswith("split"):
+                    # Hack for handling split
+                    # Split returns a Sequence that should be treats as a single
+                    # value. So we wrap it into a tuple.
+                    # TODO(justinchuby): Find a more general solution
+                    flattened_torch_outputs = (flattened_torch_outputs,)
+
                 function_output = self.function_executor(flattened_torch_outputs)(
                     onnx_function, input_onnx, kwargs_onnx
                 )
+                # Finally we re-flatten everything
+                # TODO: add pytree structure comparison.
+                flattened_torch_outputs, _ = pytree.tree_flatten(torch_output)
```

Comment:
same line at 1133, is it by design?

Reply:
Yes, because 1139 changed the shape for the function_executor. I should probably rename it